### PR TITLE
10230 multiple partners

### PR DESCRIPTION
**Pull request by roseeichelmann**
## Associated issues

This PR allows users to select multiple coordination partners for a recommendation instead of just one. It required restructuring the back-end database using a many to many relationship between partners and recommendations.

Following this PR, coordination partner information will need to be migrated from the `recommendations` table to the new `recommendations_partners` table in issue [#10462](https://github.com/cityofaustin/atd-data-tech/issues/10462)

Closes cityofaustin/atd-data-tech/issues/10230

## Testing
**URL to test:** 

This can be tested locally or with the Netlify preview: https://deploy-preview-1110--atd-vze-staging.netlify.app/#/crashes

**Steps to test:**
1. Click on Crashes tab.
2. Filter crashes by ATD death count (Recommendations section only shows up on crashes that had a fatality). Click on any crash with a fatality.
4. Scroll to Fatality Review Board Recommendations section. Add multiple partners, remove partners, refresh, make sure your changes stick.
5. You can also play around with the rest of the section to make sure everything works together.

---
#### Ship list
- [x] Code reviewed
- [x] Product manager approved

### CODE REVIEW

File: atd-vze/src/views/Crashes/Recommendations/RecommendationMultipleSelectDropdown.js [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+  options,
+  onOptionClick,
+  onOptionRemove,
+  partners,
+  fieldConfig,
+  field,
+}) => {
+  // Trigger mutation call to add partner upon click
+  const handleOptionClick = selectedItem => {
+    console.log(selectedItem);
```
Comment (frankhereford) [inline atd-vze/src/views/Crashes/Recommendations/RecommendationMultipleSelectDropdown.js:15]: Would it be alright to remove this `console.log()` and any others that are tagging along?

**Review by frankhereford — APPROVED** [review]
partner_kid

Rose, this looks and works great! I left a minor comment below, but otherwise, this is ready to roll I believe. Thank you for researching and finding a good multiselect component. This one is a winner. 

When you merge this, would you mind finding a way to make sure that a note gets into the release notes or the next release-party agenda to remind everyone about the required database schema changes? Thanks!!

File: atd-vze/src/views/Crashes/Recommendations/RecommendationMultipleSelectDropdown.js (hunk 1 above)
Comment (roseeichelmann) [reply atd-vze/src/views/Crashes/Recommendations/RecommendationMultipleSelectDropdown.js:15]: oops thanks for catching that! i literally just added that then forgot to remove it before i pushed 🤦

**Review by patrickm02L — APPROVED** [review]
This is a really elegant solution! Everything is very cohesive and flows well. Well Done! 🤘

https://user-images.githubusercontent.com/5566727/196434694-4b6a673d-9509-4257-bd08-f045b40faf7b.mp4

**Pull request merged**